waspc/run.ps1

```diff
@@ -0,0 +1,38 @@
+# This script defines common commands used during building / developing `waspc` project
+# and makes it easy to run them.
+# PSVersion 5.1 or higher is required.
+
+param(
+    [string]$Command = ""
+)
+
+$PROJECT_ROOT = Split-Path -Parent $MyInvocation.MyCommand.Path
+$REPOSITORY_ROOT = Split-Path -Parent $PROJECT_ROOT
+    
+# Building
+$WASP_PACKAGES_COMPILE = Join-Path $PROJECT_ROOT "tools\install_packages_to_data_dir.bat"
+$BUILD_HS_CMD = "cabal build all"
+$BUILD_ALL_CMD = "$WASP_PACKAGES_COMPILE -and $BUILD_HS_CMD"
+$RUN_CMD="cabal --project-dir=${PROJECT_ROOT} run wasp-cli -- $Args"
+
+switch ($Command) {
+    "build" {
+        Invoke-Expression $BUILD_HS_CMD
+    }
+    "build:all" {
+        Invoke-Expression $BUILD_ALL_CMD
+    }
+    "wasp-cli" {
+        Invoke-Expression $RUN_CMD
+    }
+    Default {
+        Write-Host "USAGE"
+        Write-Host "  run <command>"
+        Write-Host ""
+        Write-Host "COMMANDS"
+        Write-Host "  build             Builds the Haskell project."
+        Write-Host "  build:all         Builds the Haskell project + all sub-projects (i.e. TS packages)."
+        Write-Host "  wasp-cli <args>   Runs the dev version of wasp executable while forwarding arguments."
```

Comment:
Will indeed argument forwarding happen? That is something that happens by default in Powershell scripts?

Reply:
Yes, The script defines the first positional argument as `$Command` and `$Args` refers to the rest of arguments by default ([source](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_automatic_variables?view=powershell-5.1#args)).